gems/aws-sdk-core/CHANGELOG.md

```diff
@@ -1,6 +1,8 @@
 Unreleased Changes
 ------------------
 
+* Issue - Update `:response_target` plugin error handling
```

Comment:
I would expand on this. What GitHub issue does it solve, or, what kind of error does this address? Helps users identify if they need to pick up this upgrade in particular.